docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_news.yml

```diff
@@ -0,0 +1,143 @@
+id: yukon_migrate_news
+label: Yukon Migrate News Nodes Translations
+migration_tags:
+    - Drupal 7
+    - Content
+    - News
+migration_group: legacy
+source:
+    plugin: d7_node
+    node_type: news
+    constants:
+        default_text_format: full_html
+destination:
+    plugin: entity:node
+    default_bundle: news
+process:
+    uid: node_uid
+    status:
+        plugin: skip_on_value
+        method: row
+        not_equals: true
+        value: 1
+        source: status
+    langcode:
+        plugin: default_value
+        source: language
+        default_value: "en"
+    created: created
+    changed: changed
+    promote: promote
+    sticky: sticky
+    revision_log: log
+    title: title
+    field_page_description/value: field_page_description/0/value
+    field_page_description/format: constants/default_text_format
+    body/value: body/0/value
+    body/format: constants/default_text_format
+    field_contact/value: field_contact/0/value
+    field_contact/format: constants/default_text_format
+    field_backgrounder/value: field_backgrounder/0/value
+    field_backgrounder/format: constants/default_text_format
+    field_news_release: field_news_release_
+
+    # quotes
+    pseudo_news_quote:
+        plugin: sub_process
+        source: field_news_quote
+        process:
+            paragraph:
+                plugin: migration_lookup
+                source: value
+                migration: yukon_migrate_quotes
+                no_stub: true
+
+    field_news_quote:
+        plugin: sub_process
+        source: '@pseudo_news_quote'
+        process:
+            target_id: 'paragraph/0'
+            target_revision_id: 'paragraph/1'
+
+    # quick facts
+    pseudo_quick_facts:
+        plugin: sub_process
+        source: field_quick_facts
+        process:
+            paragraph:
+                plugin: migration_lookup
+                source: value
+                migration: yukon_migrate_quick_facts
+                no_stub: true
+
+    field_quick_facts:
+        plugin: sub_process
+        source: '@pseudo_quick_facts'
+        process:
+            target_id: 'paragraph/0'
+            target_revision_id: 'paragraph/1'
+
+    # charts
+    pseudo_field_charts:
+        plugin: sub_process
+        source: field_charts
+        process:
+            paragraph:
+                plugin: migration_lookup
+                source: value
+                migration:
+                    - yukon_migrate_charts
```

Comment:
please use the same syntax for simple migrations like everywhere 
`migration: yukon_migrate_charts`

Reply:
Fixed